packages/stryker/src/reporters/EventRecorderReporter.ts

```diff
@@ -33,9 +33,8 @@ export default class EventRecorderReporter implements StrictReporter {
     return this._baseFolder;
   }
 
-
-  private writeToFile(index: number, methodName: keyof Reporter, data: any) {
-    let filename = path.join(this.baseFolder, `${this.format(index)}-${methodName}.json`);
+  private writeToFile(methodName: keyof Reporter, data: any) {
+    let filename = path.join(this.baseFolder, `${this.format(this.index++)}-${methodName}.json`);
```

Comment:
It doesn't matter if it's before or after. As long as we get unique file names alphabetically ordered, i'm happy